src/rez/cli/pkg-cache.py

```diff
@@ -70,7 +78,17 @@ def add_variant(pkgcache, uri, opts):
         print("No such variant: %s" % uri, file=sys.stderr)
         sys.exit(1)
 
-    destpath, status = pkgcache.add_variant(variant, force=opts.force)
+    if opts.pkg_cache_mode == "async":
+        cache_mode = True
+    elif opts.pkg_cache_mode == "sync":
+        cache_mode = False
```

Comment:
```suggestion
        cache_mode = False
    elif opts.pkg_cache_mode == "sync":
        cache_mode = False
```
Umm, I think this would be the right thing to do. If it's async, we don't want to wait (so False).